include/dsVideoPort.h

```diff
@@ -660,6 +660,58 @@ dsError_t dsSetForceDisable4KSupport(intptr_t handle, bool disable);
  */
 dsError_t dsGetForceDisable4KSupport(intptr_t handle, bool *disable);
 
+/**
+ * @brief Enables/Disables ALLM mode for HDMI output video port.
+ *
+ * For source devices, this function enables or disables the ALLM mode for specified HDMI output video port.
```

Comment:
Continuing the first statement.  Also, mention "Enabling or disabling should follow the HDMI 2.1 Specification.  ALLM bit on HF-VSDB should be considered before setting ALLM_Mode bit on the HF-VSIF"